code/_onclick/hud/fullscreen.dm

```diff
@@ -63,6 +63,7 @@
 	. = ..()
 	if(!same_z_layer)
 		relayer_fullscreens()
+		update_client_colour()
```

Comment:
if I move up does my application to the lower z stick around?

Reply:
The answer was yes